agent/src/main/java/kanela/agent/util/conf/KanelaConfiguration.java

```diff
@@ -56,16 +56,25 @@
     @Getter(AccessLevel.PRIVATE)
     Config config;
 
-    private static class Holder {
-        private static final KanelaConfiguration Instance = new KanelaConfiguration();
```

Comment:
`KanelaConfiguration` is no longer a singleton.... for now I can live with this, but only for no

Reply:
Yeah man... I'm really not proud of this part of the PR.. I did check to ensure that this will work as expected so we are safe at the moment, but we should rethink how all components that require configuration are to be instantiated now that it is possible to reload Kanela and it might happen that after reloading there will be a different configuration :confused:, that's for another day, though.